virttest/libvirt_xml/vm_xml.py

```diff
@@ -743,7 +743,8 @@ def set_vm_vcpus(vm_name, vcpus, current=None, sockets=None, cores=None,
             # is set true and not have all topology definition, it would fail
             # let us leave it to the user give proper values
             if (topology or add_topology) and (sockets or cores or threads):
-                vmcpu_xml = VMCPUXML()
+                # Only operate topology tag, other tags doesn't change
+                vmcpu_xml = vmxml['cpu']
```

Comment:
@cuzhang This will fail  if the guest xml does not have cpu attribute in the first place,  pls add check and if not lets fallback to `vmcpu_xml = VMCPUXML()`

Reply:
@sathnaga please help to check again for this change